io_scene_godot/converters/armature.py

```diff
@@ -36,7 +67,7 @@ def __init__(self, bone_id, name, parent_id):
         self.rest = mathutils.Matrix()
         self.pose = mathutils.Matrix()
         self.enabled = True
-        self.bound_children = None
+        self.bound_children = BoundChildren()
```

Comment:
Instead of creating a specific "BoundChildren" class, I think it could be represented with a structures.Array